meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -460,6 +494,11 @@ python __anonymous() {
         part_info["mount_opts"] = _process_var_with_default(d, part_name, "MOUNT_OPTS")
         part_info["fs_freq"] = _process_var_with_default(d, part_name, "FS_FREQ")
         part_info["fs_passno"] = _process_var_with_default(d, part_name, "FS_PASSNO")
+        part_info["fs_part_numbers"] = []
```

Comment:
Can you add a comment about the reason why we are doing this computation?

Reply:
Mmmm. I'd rather not.

The reasons for the computation are:
1. We'll need it later when telling U-Boot where to find the FIT image to use.
2. It's useful for debug.

There aren't comments for every other field of `part_info`, and I don't think "fs_part_numbers" is special enough to warrant a comment. If you don't think my reasons for having the field are good enough then I should get rid of the field rather than add a comment... Although I'd rather keep it.